R/tidy-upkeep.R

```diff
@@ -111,6 +111,35 @@ if (year <= 2022) {
     )
   }
 
+  if (year <= 2023) {
+    bullets <- c(bullets,
+     "2023",
+     "",
+     "Posit updates:",
+     "",
+     todo('Update copyright holder in DESCRIPTION: `person(given = "Posit, PBC", role = c("cph", "fnd"))`'),
+     todo("Double check license file uses '[package] authors' not RStudio"),
+     todo("Update email addresses *@rstudio.com -> *@posit.co"),
+     todo("`usethis::use_tidy_coc()`"),
+     "",
+     "pkgdown:",
+     "",
+     todo("Update pkgdown site using instructions at <https://tidytemplate.tidyverse.org>"),
+     todo("Ensure pkgdown `development` is `mode: auto` in pkgdown config"),
+     todo("Submit PR [here](https://github.com/rstudio/aws-main/tree/main/zones) adding your site to the appropriate domain (eg., r-lib, tidyverse, tidymodels) and set url in GitHub Settings > Pages > Custom Domain"),
+     todo("Re-publish released site; see [How to update a released site](https://pkgdown.r-lib.org/dev/articles/how-to-update-released-site.html)"),
+     "",
+     todo("Modernize citation files; see updated `use_citation()`"),
+     todo("Update logo (https://github.com/rstudio/hex-stickers)"),
+     todo('Use `pak::pkg_install("org/pkg") in README'),
+     todo("Consider running `use_tidy_dependencies()`"),
```

Comment:
I'm only moderately familiar with this - would this be in addition to, or in place of `use_tidy_dependencies()`?

Reply:
One of the things `use_tidy_dependencies()` does is to place a standalone file that implements purrr-like functions. And we've now formalized this sort of maneuver into the new function `use_standalone()` (which can also be called directly, more generally). This also introduced new conventions around file names and tracking the dependencies between standalone files, etc.

So it would be good to publicize this within our group.

Reply:
Basically, new calls to `use_tidy_dependencies()` will use the new approach.

And folks with other so-called "compat" files lying around could conceivably want to freshen them up.

Reply:
Ok, that makes sense. Thanks!